webapp/src/lib/dialog-transaction.html

```diff
@@ -0,0 +1,20 @@
+<div class="mat-body">
+    <h1>{{data.title}}</h1>
+    <div #main class="window" *ngIf="!error">
+        <img src="assets/imgs/client.png" class="client"/>
+        <img src="assets/imgs/cothority.png" class="cothority"/>
+
+        <div class="progress-box">
+            <div [style.width.%]="percentage" class="progress-bar">
+                {{text}}
+            </div>
+        </div>
+    </div>
+    <div *ngIf="error">
+        <div class="error" >Error: {{ error }}</div>
```

Comment:
So yeah, let's choose the syntax we want for name reference in templates.
From what I read, the `{{ var }}` is used in the reference and emphasize the visibility of the reference. `{{var}}` is also used, as a more concise form.
Mixing both make it a bit harder to read. WDYT?

Reply:
Oh my - I'm always using <CMD><OPTION>L in IntelliJ and just go with whatever he proposes to me...

Let's go with `{{ error }}` - and we should find a formatter for the html, too...